apps/Standalone/src/App.tsx

```diff
@@ -50,8 +51,10 @@ const DataMapperV2 = () => {
 
 const TemplatesStandalone = () => {
   return (
-    <Provider store={templateStore}>
-      <TemplatesStandaloneDesigner />
-    </Provider>
+    <ReactQueryProvider>
```

Comment:
ReactQueryProvider exists in TemplatesStandaloneDesigner already

Reply:
will fix it.